se/__init__.py

```diff
@@ -300,14 +300,15 @@ def get_xhtml_language(xhtml: str) -> str:
 
 	return language
 
-def get_firefox_path() -> str:
+def get_firefox_path() -> Path:
 	"""
 	Get the path to the local Firefox binary
 	"""
 
-	try:
-		firefox_path = Path(shutil.which("firefox"))
-	except Exception:
+	which_firefox = shutil.which("firefox")
```

Comment:
Curious why you changed this to create a new variable--how is this an improvement over the previous way?

Reply:
The intention is to resolve the type conflict detected my mypy. The `Path()` constructor does not accept `None` as an argument. The `shutil.which()` function returns `None` or a `str` (type `Union[None,str]`) so mypy complains. By filtering out the `None` result with the if statement, the type effectively becomes `str` so the conflict is resolved.

There are also the style arguments: 1) it's generally considered poor style to catch `Exception` rather than a more specific error because it masks unexpected errors, and 2) the condition check is arguably more explicit - we don't expect any errors, just a possible None from `shutil.which()`.